lib/classes/calculator.rb

```diff
@@ -0,0 +1,54 @@
+
+data = [
+  4,
+  "6",
+  { result: "Deset"}
+]
+
+
+puts "#{data[0]} + #{data[1]} = #{data[2][:result]} "
+
```

Comment:
this should be in other file, maybe inside run.rb

lib/classes/calculator.rb should contain only definition of class